phpstan.neon.dist

```diff
@@ -5,3 +5,7 @@ parameters:
     - src
     - tests
   level: 9
+  ignoreErrors:
+    - "#Method [a-zA-Z0-9\\_\\\\:\\(\\)]+ has parameter \\$[a-zA-Z0-9_]+ with no value type specified in iterable type array#"
+    - "#Method [a-zA-Z0-9\\_\\\\:\\(\\)]+ return type has no value type specified in iterable type array#"
+    - "#Property [a-zA-Z0-9\\$\\_\\\\:\\(\\)]+ type has no value type specified in iterable type array#"
```

Comment:
These specific errors could be ignored using `checkMissingIterableValueType: false` ([docs](https://phpstan.org/config-reference#vague-typehints)):

```suggestion
  checkMissingIterableValueType: false
```